src/solve1.F

```diff
@@ -416,6 +416,7 @@ subroutine solve1(nstep,rbufsz,num_soil_layers,                  &
       else
         tqnudge = .false.
       endif
+      tqnudge = .false.
```

Comment:
So this setting will always set `tqnudge` to `False`. Is this intended? If so, we probably should remove the `if` block above.